src/services/destination/cdkV2Integration.ts

```diff
@@ -109,6 +122,15 @@ export default class CDKV2DestinationService implements IntegrationDestinationSe
           try {
             const doRouterTransformationResponse: RouterTransformationResponse[] =
               await processCdkV2Workflow(destinationType, destInputArray, tags.FEATURES.ROUTER);
+
+            stats.increment('event_router_success', {
```

Comment:
I think this can also be `event_transform_success` as we have feature, implementation that can differentiate about the transformation nature